charts/alertmanager/Chart.yaml

```diff
@@ -6,7 +6,7 @@ icon: https://raw.githubusercontent.com/prometheus/prometheus.github.io/master/a
 sources:
   - https://github.com/prometheus/alertmanager
 type: application
-version: 0.6.0
+version: 0.6.1
```

Comment:
New feature should raise minor version.

Reply:
Sorry somehow I forgot to commit it. I will fix it now thanks!